sqlserver/tests/test_high_cardinality.py

```diff
@@ -87,7 +88,7 @@ def _run_queries(idx):
 
 @high_cardinality_only
 @pytest.mark.skip(reason='skip until the metrics query is improved')
-@pytest.mark.parametrize('job', ['query_metrics', 'query_activity'])
+@pytest.mark.parametrize('job', ['query_metrics', 'query_activity', 'procedure_metrics'])
```

Comment:
unless we add support for proc stuff in these high cardinality tests, i think we can leave this out for now